lostparticles/lostparts2mesh.py

```diff
@@ -0,0 +1,67 @@
+
+import sys
+import numpy as np
+
+uvw_tag_name = "RAY_DIR"
+output_filename = "lost_particle_locations.h5m"
+#open the mcnp output file
+file = open(sys.argv[1], 'rb')
+#start with five lines in our line_cache
+line_cache = [file.readline(),file.readline(),file.readline(),file.readline(),file.readline(),file.readline()]
+#dictionary for holding lost particle info
+lost_part_dict = {}
+for line in file:
+    #add line to cache
+    line_cache.append(line)    
+    #line_cache should always be size six here
+    assert(7 == len(line_cache))
+    #check for lost particle indicator substring
+    if "no intersection found in subroutine track" in line_cache[0]:
+        #lost particle add last known location and number to dictionary
+        #need to adjust parsing if more than 100 are lost
+        split_key_line = line_cache[0].split()
+        key = split_key_line[4] if len(lost_part_dict) < 100 else split_key_line[3].split('.')[-1]
+        xyz = line_cache[-2].split()[2:5]
+        uvw = line_cache[-1].split()[3:6]
+        #add to lost particle dict
+        lost_part_dict[key] = xyz + uvw
+    #remove line from cache
+    line_cache.pop(0)
+    
+#print out some info for user verification
+print("Found" ,len(lost_part_dict), "lost particles.")
+print(lost_part_dict["no"])
+
+#write the data to a mesh
+try:
+    from pymoab import core,types
+    mbi = core.Core()
+    print("Using pymoab to generate mesh file....")
+    ray_tag_handle = mbi.tag_get_handle(uvw_tag_name, 3, types.MB_TYPE_DOUBLE, True)
+    #vertices to the MOAB instance
+    verts = mbi.create_vertices(np.array([float(i) for l in lost_part_dict.values() for i in l[:3]],dtype='float64'))
+    #and now tag the verices with the lost particle directions
```

Comment:
Sounds good. This is what I get from the mbconvert-ed vtk file. These can be written directly as vtks too of course, but I thought I'd try to stick with the native MOAB format.
![screenshot from 2016-04-05 15-38-04](https://cloud.githubusercontent.com/assets/4563941/14297501/a750c2c8-fb44-11e5-9287-7ab0b309e048.png)